amqp-common/src/main/java/uk/nhs/adaptors/amqp/task/TaskDefinition.java

```diff
@@ -0,0 +1,20 @@
+package uk.nhs.adaptors.amqp.task;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@SuperBuilder
+@EqualsAndHashCode
+public abstract class TaskDefinition {
+    private final String taskId;
```

Comment:
do we need all those fields? maybe it is a better idea to start adding them once we need them?